protocol/synthetix/contracts/modules/core/IssueUSDModule.sol

```diff
@@ -84,9 +84,11 @@ contract IssueUSDModule is IIssueUSDModule {
         // If the resulting debt of the account is greater than zero, ensure that the resulting c-ratio is sufficient
         (, uint256 collateralValue) = pool.currentAccountCollateral(collateralType, accountId);
         if (newDebt > 0) {
+            uint256 minIssuanceRatioD18 = pool.issuanceRatioD18[collateralType];
```

Comment:
Would it be bad practice to just put this on line 91 instead of creating a new variables?